x-pack/plugins/security_solution/public/sourcerer/experimental/use_unstable_security_solution_data_view.ts

```diff
@@ -19,6 +23,16 @@ export const useUnstableSecuritySolutionDataView = (
   _scopeId: SourcererScopeName,
   fallbackDataView: SelectedDataView
 ): SelectedDataView => {
-  // TODO: extend the fallback state with values computed using new logic
-  return IS_EXPERIMENTAL_SOURCERER_ENABLED ? fallbackDataView : fallbackDataView;
+  const dataView: SelectedDataView = useSelector(sourcererAdapterSelector);
+
+  const dataViewWithFallbacks: SelectedDataView = useMemo(() => {
+    return {
+      ...dataView,
```

Comment:
It looks like this instance of dataView is not consistent with the values selected by the user in the UI.

I have added a demo video in my comment above.

Reply:
hey, data view creation is not supported currently. also, the returned data view is not complete yet.